docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
 title: API Reference
 ---
 
-## Commands
+## Read Model Connector Interface
 
-### Command HTTP API
+The table below lists functions that a custom Read Model's connector should implement.
 
-A command can be sent using HTTP API.
+| Function Name             | Description                                                                                |
+| ------------------------- | ------------------------------------------------------------------------------------------ |
+| [connect](#connect)       | Initializes a connection to a storage.                                                     |
+| [disconnect](#disconnect) | Closes the storage connection.                                                             |
+| [drop](#drop)             | Removes the Read Model's data from storage.                                                |
+| [dispose](#dispose)       | Forcefully disposes all unmanaged resources that Read Models served by this connector use. |
 
-For instance, to create a new list in the shopping list app:
+### connect
+
+Initializes a connection to a storage. An implementation should return a store object.
```

Comment:
"An implementation" any implementation?

Reply:
Yes, we expect users to implement their own versions of this function.